stdlib/public/SDK/ObjectiveC/ObjectiveC.swift

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if _runtime(_ObjC)
```

Comment:
Do we try to build the ObjectiveC module on non-ObjC-interop platforms? I didn't see a CMakeLists.txt change to that effect.

Reply:
I can revert this if it’s unneeded. Part of the debugging cycle for this patch was building it on Darwin simulating the Linux situation so that I could iterate on it with Xcode; it may be a leftover from that.